data/en/words/entities.locations.sing

```diff
@@ -1,6 +1,6 @@
 Africa.l Agreste.l Alabama.l Alberta.l Aleppo.l 
 Antarctic.l Antigua.l Arctic.l Arizona.l 
-Arkansas.l Arusha.l Asia.l Aspen.l Assyria.l Atlanta.l 
+Arkansas.l Arusha.l Asia.l Aspen.l Assyria.l Atlanta.l Avonlea.l
```

Comment:
Do you think that we need to contribute the changes like this to mainstream Link Grammar project?